### DISCUSSION TITLE
OpenResty 升级失败.1Panel-openresty-fppQ 容器不存在

### DISCUSSION THREAD

**Discussion by AtlantisDe** [1Panel-dev/1Panel#5616] [Q&A]
### OpenResty 升级失败.1Panel-openresty-fppQ 容器不存在 

- 当前版本:版本：1.21.4.3-3-1-focal
- 1panel: v1.10.11-lts
- 想升级到的版本:  高点版本.

```c#

1Panel-openresty-fppQ 容器 看这字面是不是PHP ,的容器环境?

我目前没有安装这个PHP 环境,就导致无法升级成功吗?

```
![image](https://github.com/1Panel-dev/1Panel/assets/39150678/8ae6b1ec-89b3-4281-8c77-95edd8680777)

![image](https://github.com/1Panel-dev/1Panel/assets/39150678/a4068f38-d855-493d-9321-dddc7db57f8b)

![image](https://github.com/1Panel-dev/1Panel/assets/39150678/cc0d41c5-493b-49ad-962d-c6efdb71870f)

升级后 容器 都没了?

**Comment by wanghe-fit2cloud** [comment]
进入 `主机` - `终端`，手动执行 docker pull 命令拉取一下 OpenResty 目标版本镜像。
> `容器` - `配置` 页面支持设置镜像加速。
